rmf_task/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 ## Changelog for package rmf_task
 
+2.1.0 (2022-XX-YY)
```

Comment:
```suggestion
2.1.0 (2022-05-05)
```

Reply:
Apologies, I noticed a mistake in my review. I believe we will need this change log in `rmf_task_sequence` instead of `rmf_task`.

Just checked in with @marcoag, it sounds like we will need all packages in this repo to have mentioned this minor version, even if there were no changes to the packages.